pages/content-api/filters.mdx

```diff
@@ -140,76 +140,78 @@ If you are wondering how you can combine multiple queries, then you may know tha
 </Tabs.Tab>
 <Tabs.Tab>
 ```tsx
-// TypeScript code is coming soon...
+await unbody.get.googleDoc.where(({ GreaterThan, LessThan, And }) => {return And(({ size: GreaterThan(100), createdAt: LessThan("2023-01-01T00:00:00Z") }))}).select('title').exec()
```

Comment:
the code format is not good. 
we need to use indentation and break each command in one new line.